docs/dictionary/control_st/if.lcdoc

```diff
@@ -4,9 +4,30 @@ Type: control structure
 
 Syntax: if <condition> then <statementList> [else <elseStatementList>]
 
+Syntax:if <condition> then
+   <statementList>
+ [else if <condition> then
+   <elseifStatementList>]
+ [else 
+   <elseStatementList>]
+end if
+
+
 Summary:
-<execute|Executes> a list of <statement|statements> if a condition is
-true. 
+Evaluates a <condition|conditional expression> to determine if it is true
+and if so <execute|executes> a subsequent <statement> or <statementlist|statement list>.
+
+If the <conditional expression|condition> is false and the <if> <control structure> contains
+an <else> keyword the <statement> or <statementlist|statement list> following the <else>
+<keyword> is executed.
+
+The <if> <control structure> may contain one or more <else if> <keyword|keywords> which have a
+<conditional expression|condition>. If the <conditional expression|condition> one of these evaluates to be
```

Comment:
There's a couple more `<conditional expression|condition>` that need switching round